src/main/java/io/vlingo/http/BinaryBody.java

```diff
@@ -0,0 +1,43 @@
+// Copyright © 2012-2018 Vaughn Vernon. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+
+package io.vlingo.http;
+
+import java.nio.charset.StandardCharsets;
+
+public class BinaryBody implements Body {
+
+  public final byte[] binaryContent;
+
+  @Override
+  public String content() {
```

Comment:
In this case maybe you could return a base64 `String`. At least it fulfills the contract for a binary body to `String` per the actual underlying implementation.

Reply:
👍, done.